src/components/content-tab-assistant.tsx

```diff
@@ -49,6 +50,18 @@ const UsageLimitReached = () => {
 	);
 };
 
+const OfflineModeView = () => {
+	// Determine if the reset is today
```

Comment:
Is this comment specific for this component?

Reply:
Removed in https://github.com/Automattic/studio/pull/271/commits/0227be1ad1291dfc0ba5ffe550f19732e35ffda8
That should only be present in the `UsageLimitReached` component.